src/utils/merge.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import assign from "./assign";
+import isNonEmptyArray from "./isNonEmptyArray";
+/**
+ * Returns an object containing keys from all passed in values
+ * @param {*} values
+ * @returns {Object}
+ */
+export default values => {
+  if (isNonEmptyArray(values)) {
+    return values.reduce((acc, e) => assign(acc, e), {});
+  }
+  return {};
+};
```

Comment:
@ninaceban My understanding is that this does basically the same thing:

```
assign({}, ...values);
```

except that it doesn't do the `isNonEmptyArray(values)` check, which shouldn't be necessary where you were using it anyway (https://github.com/adobe/alloy/pull/398/commits/604c60a5381d1104476f608ece3bc7161ca8d010#diff-f13c18241f022f5285cf29e5d815b506R109-R110). Did you try that and it didn't work or is this one of those doesn't-work-in-IE things?

Reply:
`assign({}, ...values);` - this will work only if I pass values to a `flatMap`

Reply:
I'll remove unused utils